aap_policy_examples/project_scm_branch.rego

```diff
@@ -0,0 +1,32 @@
+package aap_policy_examples
```

Comment:
this was also in another PR https://github.com/TheRealHaoLiu/example-opa-policy-for-aap/pull/16 

lets remove it from this PR